tests/test_rechunk.py

```diff
@@ -207,7 +224,13 @@ def test_rechunk_array(
 @pytest.mark.parametrize("dtype", ["f4"])
 @pytest.mark.parametrize("max_mem", [25600000])
 @pytest.mark.parametrize(
-    "target_chunks", [(200, 8000), (800, 8000), (8000, 200), (400, 8000),],
+    "target_chunks",
+    [
+        (200, 8000),
+        (800, 8000),
+        (8000, 200),
+        (400, 8000),
+    ],
```

Comment:
Is this necessary? This seems to be just formatting

Reply:
Um yes, that seems to be some automatic formatting. I will make sure to properly reformat for the final PR.